staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/helper/helpers.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
who uses these helpers? Are they strictly internal to the component? If yes, we should merely move them out of the pkg/apis package.

Reply:
if they are only used by the controller, move the helpers there somewhere.